src/main/java/stsjorbsmod/cards/cull/Panic.java

```diff
@@ -0,0 +1,50 @@
+package stsjorbsmod.cards.cull;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.IntangiblePlayerPower;
+import com.megacrit.cardcrawl.vfx.combat.FlashAtkImgEffect;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.cards.CustomJorbsModCard;
+import stsjorbsmod.characters.Cull;
+import stsjorbsmod.patches.EphemeralField;
+
+public class Panic extends CustomJorbsModCard {
+    public static final String ID = JorbsMod.makeID(Panic.class);
+
+    private static final CardRarity RARITY = CardRarity.COMMON;
+    private static final CardTarget TARGET = CardTarget.SELF;
+    private static final CardType TYPE = CardType.SKILL;
+    public static final CardColor COLOR = Cull.Enums.CULL_CARD_COLOR;
+
+    private static final int COST = 0;
+    private static final int INTANGIBLE_AMOUNT = 1;
+    private static final int MAXHP_LOSS = 3;
+    private static final int UPGRADE_MAXHPLOSS = -2;
+
+    public Panic() {
+        super(ID, COST, TYPE, COLOR, RARITY, TARGET);
+        magicNumber = baseMagicNumber = INTANGIBLE_AMOUNT;
+        urMagicNumber = baseUrMagicNumber = MAXHP_LOSS;
+        EphemeralField.ephemeral.set(this, true);
+    }
+
+    @Override
+    public void use(AbstractPlayer p, AbstractMonster abstractMonster) {
+        addToBot(new ApplyPowerAction(p, p, new IntangiblePlayerPower(p, magicNumber)));
+        AbstractDungeon.effectList.add(new FlashAtkImgEffect(p.hb.cX,p.hb.cY, AbstractGameAction.AttackEffect.BLUNT_LIGHT));
+        AbstractDungeon.player.decreaseMaxHealth(urMagicNumber);
```

Comment:
I think we need to create a new LoseMaxHPAction for this to use rather than calling `decreaseMaxHealth` directly here because the power Phantasm deals damage on gaining intangible; if that damage ends the fight, we're going to want the max HP loss to be prevented (via the usual path where clearPostCombatActions would do so in the damage action).

Reply:
(Possibly it would be good for the attack effect to be moved into that action as well)

Reply:
I actually needed a `DecreaseMaxHpAction` for a separate purpose earlier tonight, so that's now pushed to master; you can merge master and then re-use that action rather than implementing a new one!